boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -148,26 +148,28 @@ loads from SHD SPI.
 Peripheral Routing Jumpers
 --------------------------
 
-Each column of the following table illustrates how to enable UART0, JTAG,
+Each column of the following table illustrates how to enable SWD,
 PVT SPI, SHD SPI and LED0-2 respectively.
 
 +---------+--------+-----------+----------+---------+
-| JP68    | JP9    | JP38      | JP98     | JP41    |
-| (UART0) | (JTAG) | (PVT SPI) | (SHD SPI)| (LED0-2)|
+| JP9     | JP9    | JP38      | JP98     | JP41    |
```

Comment:
Should these both be JP9?

Reply:
Correct both features are controlled from JP9